### PR TITLE
ansible : deploy jenkins slave(s) only

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

File: scripts/ci/ansible/roles/jenkins_master/tasks/main.yml [hunk 1]
```diff
@@ -5,13 +5,25 @@
   with_items:
     - id_rsa
     - id_rsa.pub
+  tags:
```
Comment (masco-enovance) [inline scripts/ci/ansible/roles/jenkins_master/tasks/main.yml]: nit: how about renaming the parent folder from jenkins_master to jenkins?

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

File: scripts/ci/ansible/inventory.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+localhost
```
Comment (safchain) [inline scripts/ci/ansible/inventory.yml]: need to remove this file :)

Comment (nplanel) [reply scripts/ci/ansible/inventory.yml]: I forgot to ask @lebauce if it was a tricks

Comment (safchain) [reply scripts/ci/ansible/inventory.yml]: the inventory does have to be submitted, only used during the deployment

**Comment by nplanel** [comment]
run skydive-go-fmt

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-selenium-tests

File: scripts/ci/ansible/roles/jenkins_master/tasks/main.yml (hunk 1 above)
Comment (nplanel) [reply scripts/ci/ansible/roles/jenkins_master/tasks/main.yml]: jenkins_master role is only used to deploy jenkins master, so let the directory named like this

**Comment by nplanel** [comment]
run skydive-unit-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
rerun functional-tests-backend-elasticsearch

**Comment by nplanel** [comment]
rerun functional-tests-backend-elasticsearch

**Pull request merged**